build/Dockerfile.s390x

```diff
@@ -16,11 +16,11 @@
 
 FROM alpine as builder
 
-RUN wget -O /qemu-s390x-static https://github.com/multiarch/qemu-user-static/releases/latest/download/qemu-s390x-static
+RUN wget -O /qemu-s390x-static https://github.com/multiarch/qemu-user-static/releases/v5.2.0-2/download/qemu-s390x-static
```

Comment:
I tested that the wget for the url is not working, the correct url should be: 
https://github.com/multiarch/qemu-user-static/releases/tag/v5.2.0-2/download/qemu-s390x-static ?